api/migrations/versions/1c3fda18bad6_.py

```diff
@@ -0,0 +1,413 @@
+                         {'label': 'Female', 'value': 'female'},
+                         {'label': 'Transgender', 'value': 'transgender'},
+                         {'label': 'Gender variant/non-conforming', 'value': 'gender-variant/non-conforming'},
+                         {'label': 'Prefer not to say', 'value': 'prefer-not-to-say'}
+                     ])
+    demographics_q4 = Question(app_form.id, demographics.id, 'multi-choice', 'Disabilities', 'Disabilities', 4, None, description = 'We collect this information to ensure that we provide suitable facilities at our venue.',
+                      options = [
+                         {"label": "No disabilities", "value": "none"},
+                         {"label": "Sight disability", "value": "sight"},
+                         {"label": "Hearing disability", "value": "hearing"},
+                         {"label": "Communication disability", "value": "communication"},
+                         {"label": "Physical disability(e.g. difficulty in walking)", "value": "physical"},
+                         {"label": "Mental disability(e.g. difficulty in remembering or concentrating)", "value": "mental"},
+                         {"label": "Difficulty in self-care", "value": "self-care"},
+                         {"label": "Other", "value": "other"}
+                      ])
+    demographics_q5 = Question(app_form.id, demographics.id, 'short-text', 'Affiliation', 'Affiliation', 5, None, description = 'The university / institution / company you are based at')
+    demographics_q6 = Question(app_form.id, demographics.id, 'short-text', 'Department', 'Department', 6, None, description = 'The department or field of study you fall under at your affiliation')
+    
+    session.add_all([demographics_q1, demographics_q2, demographics_q3, demographics_q4, demographics_q5, demographics_q6])
+
+    about_you = Section(app_form.id, 'Tell Us a Bit About You', """
+                         Please use this section to tell us a bit more about yourself and your intentions as a future Deep Learning Indaba ambassador
+                         Take your time to fill in this section as it has the highest impact on whether or not you will be offered a place at the Indaba!""", 3)
+    session.add(about_you)
+    session.commit()
+
+    about_you_q1 = Question(app_form.id, about_you.id, 'long-text', 'Why is attending the Deep Learning Indaba 2020 important to you?', 'Enter 100 to 200 words', 1, validation_regex = '^\s*(\S+(\s+|$)){100,200}$', description = 'Enter 100 to 200 words')
+   
+    about_you_q2 = Question(app_form.id, about_you.id, 'long-text', 'How will you share what you have learned after the Indaba?', 'Enter 50 to 150 words', 2, validation_regex = '^\s*(\S+(\s+|$)){50,150}$', description = 'Enter 50 to 150 words')
+    about_you_q2.depends_on_question_id = main_q1.id
+    about_you_q2.show_for_values = ['undergraduate, masters, phd, post-doc, student at a coding academy, unemployed']
```

Comment:
This should be a list of strings. ie:
`['undergraduate', 'masters', 'phd', 'post-doc', 'student at a coding academy', 'unemployed']`
Same for questions below.